experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/magma-setup.yaml

```diff
@@ -14,11 +14,16 @@
 
 - name: Add magma user to wheel group
   user:
-    name=magma
-    groups=magma
-    append=yes
-    state=present
-    createhome=yes
```

Comment:
Default values for **state** is `present` and for **createhome** is `yes`, so removing these two.